doc/source/dev/gitwash/development_workflow.rst

```diff
@@ -195,27 +187,78 @@ Standard acronyms to start the commit message with are::
    REL: related to releasing numpy
 
 
+.. _asking-for-merging:
+
+Asking for your changes to be merged with the main repo
+=======================================================
+
+When you feel your work is finished, you can file a pull request and
+submit an optional code review. Code reviews are typically submitted when an
```

Comment:
Ah yes, read the below. I'd say something like "When you change an existing API or add a new function to the the API, please send a summary of your change with motivation to the numpy-discussion mailing list - that's the place where decisions on API changes are made."